modules/gitops-release-notes-1-3-2.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-2_{context}"]
+= Release notes for {gitops-title} 1.3.2
+
+{gitops-title} 1.3.2 is now available on {product-title} 4.6, 4.7, 4.8, and 4.9.
+
+[id="new-features-1-3-2_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.2:
+
+* Upgraded Argo CD to version *2.1.8*
+
+* Upgraded Dex to version *2.30.0*
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Previously, the {gitops-title} did not appear in the `OperatorHub` when you filtered by "Disconnected" under the *Infrastructure Features*, since the Operator did not have the related annotation set in its CSV. With this update, the {gitops-title} is annotated with "Disconnected Cluster" as an *Infrastructure Feature* link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
```

Comment:
the {gitops-title} Operator
`Disconnected` We do not use double quotes to indicate GUI features/user entered values
CSV file?

Reply:
```suggestion
* Previously, in the OperatorHub UI under the *Infrastructure Features* section, when you filtered by `Disconnected` the {gitops-title} Operator did not show in the search results, as the Operator did not have the related annotation set in its CSV file. With this update, the `Disconnected Cluster` annotation has been added to the {gitops-title} Operator as an infrastructure feature. link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
```

Reply:
the `Disconnected Cluster` - should this be `disconnected` see the [doc link](https://docs.openshift.com/container-platform/4.8/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs) provided in [the issue](https://issues.redhat.com/browse/GITOPS-1446) )